lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -182,6 +187,11 @@ class CommandInterpreterRunOptions {
   void SetSpawnThread(bool spawn_thread) {
     m_spawn_thread = spawn_thread ? eLazyBoolYes : eLazyBoolNo;
   }
+  bool GetAllowRepeats() const { return DefaultToNo(m_allow_repeats); }
```

Comment:
Nit: newline between `SetSpawnThread` and `GetAllowRepeats`?

Reply:
done